tests/files/test-curl-streamer.php

```diff
@@ -31,6 +31,13 @@ public function tearDown(): void {
 	}
 
 	public function test__init() {
+		global $wp_version;
+
+		$version = preg_replace( '/-.*$/', '', $wp_version );
+		if ( version_compare( $version, '6.4', '>=' ) ) {
+			$this->setExpectedDeprecated( 'http_api_transports' );
+		}
```

Comment:
Right now, there is no other way around this deprecation. WordPress calls `_get_first_available_transport()` from `wp_http_supports()` invoking this deprecated filter.